### ISSUE TITLE
Usage of lemur in production enviroments

### ISSUE THREAD

**Issue by adibrastegarnia** [Netflix/lemur#2871]
Hi, 

Is there any use case that shows how lemur used in production environments (public or private cloud based on k8s)? 

Thanks,
Adib

**Comment by MartijnDekkers** [comment]
...netflix?

**Comment by rohanmehto2** [comment]
Yes its hard to have a production grade lemur deployment in a k8s cluster. It would be great to have some documentation or guide to help with this.
